### PR TITLE
Created testing infrastructure

### PR DESCRIPTION
**Pull request by sgraham9319**
Function for calculating density now in the utils.R script. Can pass vector of coordinates to create a contour map of density across a specified stand.

### CODE REVIEW

File: R/utils.R [hunk 1]
```diff
@@ -71,4 +71,54 @@ defined_period_annual_growth <- function(data, begin, end){
...
+      filter(dist <= nbhd_radius) %>%
+      
+      # Calculate cross-sectional area at breast height (abh) for each tree
+      mutate(abh = pi * ((Dbh_cm / 2) ^ 2)) %>%
+      
+      # Extract number of trees and total abh within neighborhood
+      summarize(x_coord = x[coord_num],
+                y_coord = y[coord_num],
+                num_trees = n(),
+                total_abh = sum(abh))
```
Comment (arokem) [inline R/utils.R]: To get density, we'll need to divide by the area of the circle we are considering (i.e. divide by pi*100 for the default 10 meters)

Comment (sgraham9319) [reply R/utils.R]: Function now divides by the neighborhood area

**Comment by sgraham9319** [comment]
Also added a data formatting script that combines growth and neighborhood density information into a single data frame

**Comment by sgraham9319** [comment]
Edited scripts so they now all use data files with the same format (snake case column headings)

**Comment by sgraham9319** [comment]
Testing infrastructure and a test for a basic case of the nbhd_density function have been added. Please merge this so you can download the testing parts to your local repo

**Comment by arokem** [comment]
Yup. Looks good!

**Pull request merged**